### ISSUE TITLE
Ability to choose pre-defined agents

### ISSUE THREAD

**Issue by joshkyh** [rnadigital/agentcloud#6]
**Is your feature request related to a problem? Please describe.**
I can't seem to choose the agents I have pre-defined in the agents tab when starting a new session.

**Describe the solution you'd like**
When starting a new session, I would like to **choose** the agents I've predefined under agents instead of adding new agents again.

**Describe alternatives you've considered**
The alternatives considered is to ignore the agents created and start the session and (1) use the roles suggested by the roles planner or (2) re-add agents from "start a session manually" both are not ideal.

**Additional context**

"Start a session manually" should first ask us whether we want to pick agents or add agents.

<img width="922" alt="image" src="https://github.com/rnadigital/agentcloud/assets/14764966/0b3cda89-3356-43ad-8b53-7fc850da2fcd">

**Comment by ragyabraham** [comment]
@joshkyh I agree with you. This is a feature that is needed in the platform. Right now we rely on auto-team creation. However, this commit [https://github.com/rnadigital/agentcloud/commit/1cef5b9b411efea300c894dc72b4c77c33bdcbe3](https://github.com/rnadigital/agentcloud/commit/1cef5b9b411efea300c894dc72b4c77c33bdcbe3) is working towards giving users the ability to start a "manual session" where they can configure their own agents. @tomlynchRNA is working on the webapp side to enable this
